tests/backend/mongodb/test_schema.py

```diff
@@ -1,5 +1,7 @@
 import pytest
-from unittest.mock import patch
+
+from unittest import mock
+from pymongo.database import Database
```

Comment:
pep 8 import, same as above, since unitttest is in the std lib:

```python
from unittest.mock import patch

import pytest
from pymongo.database import Database
```